rpm/macros.skelcd

```diff
@@ -0,0 +1 @@
+%skelcd_datadir %{_datadir}/installation-products
```

Comment:
After thinking about it a bit more maybe we should use some more specific name to avoid possible collisions. E.g. `skelcd_path` is already used [here](https://github.com/yast/skelcd-control-Kubic/blob/master/package/skelcd-control-Kubic.spec#L133), the openSUSE [prjconfig](https://build.opensuse.org/project/prjconf/openSUSE:Factory) uses `skelcd_compat` and maybe there are some more.

So I'd rather suggest something like `skelcd_control_datadir` to have a more unique name.